### ISSUE TITLE
Add start location and end location for tags

### ISSUE THREAD

**Issue by golopot** [syavorsky/comment-parser#51]
For example

```
/**
 * @returns {number} Foo.
 */
```
Could be parsed to
```
{
    "tag": "returns",
    "type": "number",
    ...
    "range": [7, 29]
}
```
Because `@` is at position 7 and `.`  is at position 28  of the source.

**Comment by brettz9** [comment]
The stringifier might be able to use range info as well, so as to rebuild:

`/** @someTag */`

instead of:

```js
/**
 * @someTag
*/
```

**Comment by syavorsky** [comment]
this data is available since `1.0`, please see #99
